### DISCUSSION TITLE
I have just lost my free Github Copilot access.

### DISCUSSION THREAD

**Discussion by fangfufu** [community/community#87411] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

For the past year, I have had free access to Github Copilot. I believe it was through "maintainer of a popular open source project". Github has just now removed this free Copilot access from me. 

I maintain these two repositories:
- https://github.com/fangfufu/httpdirfs (694 stars)
- https://github.com/fangfufu/Linux-Fake-Background-Webcam (1.6k stars)

So who actually qualifies for free Github Copilot access? How comes I had it last year, now I have lost it? There hasn't been any circumstantial changes in terms of the repository I maintain.

**Comment by mishmanners** [comment]
Thanks for reaching out. I've passed this onto the Open Source Maintainers person who can check in on this for you.

**Comment by martinwoodward** [comment, marked as answer]
Hey @fangfufu - I'm afraid you were getting your free access as a verified member of our student community. All students and teachers who are verified with GitHub get access to GitHub Copilot as well as the maintainers of the most popular open source projects on GitHub. Your projects don't currently qualify as one of the top open source projects I'm afraid, so you'd need to purchase Copilot for Individuals by visiting https://github.com/github-copilot/signup.

**Reply by fangfufu** [reply]
Ah I see, I finished my PhD back in 2022, not knowing that the student status was still active back in 2023. I double checked the record. I re-verified the student status back in 2020. I guess that's why. 

@martinwoodward, I have to say I really do appreciate your answer. I actually filed a Github support request. From that ticket, I got a very generic response along the lines that "the popularity of my repository probably no longer meet the criteria". That answer really rubbed me the wrong way, it felt like I won the popularity contest, now I lost it. I have to say I had a bit of resentment. 

It feels better to know that I had never won the popularity contest in the first place. Now I feel this whole thing is a good learning experience. I think as a GitHub-wide improvement, it might be helpful to let the user know how they got their free co-pilot access, so people don't overestimate their abilities. :)

**Reply by martinwoodward** [reply]
Agreed. It would also help folks understand how good things like the GitHub Student Pack are.   Thanks for your understanding and thanks for being part of the student community as well as now the open source community.  We’d love to be able to give Copilot to everyone doing any open source but sadly we don’t have the capacity to do that quite yet.

**Reply by gioragutt** [reply]
Hi @martinwoodward, I have a similar issue where I had a free copilot subscription due to being an open source maintainer, and it recently ended.

I assume that this means that the repo I maintain (sindresorhus/is) is no longer considered a popular repo as per the free subscription definition? How can I verify that?
